src/embind/embind.js

```diff
@@ -2371,6 +2380,15 @@ var LibraryEmbind = {
         } else {
           proto[methodName].overloadTable[argCount-1] = func;
         }
+
+        if (classType.registeredClass.__derivedClasses) {
+          for (const derivedClass of classType.registeredClass.__derivedClasses) {
+            if (!derivedClass.constructor.hasOwnProperty(methodName)) {
+              derivedClass.constructor[methodName] = func;
```

Comment:
This doesn't consider overloads... I'm guessing I would need to?

Reply:
To match how C++ would work we probably should, but I don't think it's needed right now. It also doesn't look like overloading completely works for regular class methods right now anyway.

Adding a todo here would be good though.

Reply:
I added a todo.